x/dex/keeper/epoch.go

```diff
@@ -14,13 +14,13 @@ func (k Keeper) SetEpoch(ctx sdk.Context, epoch uint64) {
 	bz := make([]byte, 8)
 	binary.BigEndian.PutUint64(bz, epoch)
 	store.Set([]byte(EpochKey), bz)
+	ctx.Logger().Info(fmt.Sprintf("Current epoch %d", epoch))
 }
 
 func (k Keeper) IsNewEpoch(ctx sdk.Context) (bool, uint64) {
 	store := ctx.KVStore(k.storeKey)
 	b := store.Get([]byte(EpochKey))
 	lastEpoch := binary.BigEndian.Uint64(b)
 	currentEpoch := k.EpochKeeper.GetEpoch(ctx).CurrentEpoch
-	ctx.Logger().Info(fmt.Sprintf("Current epoch %d", currentEpoch))
```

Comment:
![image](https://user-images.githubusercontent.com/18161326/193434158-b84ad660-3269-4d34-a95a-1783aec5405c.png)
This is super noisy since it gets called once for each settlement check
![image](https://user-images.githubusercontent.com/18161326/193434166-db6d279b-825e-481a-a74c-0fe1818573d2.png)